tests/test_config.py

```diff
@@ -26,6 +27,19 @@ def test_load_ssl_config_verify_existing_file():
     assert context.check_hostname is True
 
 
+@pytest.mark.parametrize("config", ("SSL_CERT_FILE", "SSL_CERT_DIR"))
+def test_load_ssl_config_verify_env_file(config):
+    default_path = httpx.config.DEFAULT_CA_BUNDLE_PATH
```

Comment:
We should use a non-default and verify that it's loaded. I think there's a way to see how many certificates are loaded in an SSLContext?

Reply:
I couldn't understand this part. Could you give an example roughly?

Reply:
Basically if we don't set any `SSL_CERT_...` environment variable we choose `httpx.config.DEFAULT_CA_BUNDLE_PATH` so we can't test the behavior of `SSL_CERT_...` using that path as our expected value because there's no way to tell if the environment variable worked or we just picked the default anyways.

I suggest you use one of the auto-generated cert fixtures to test this functionality. :)

Reply:
thanks. in addition, added ssl_config.verify validation to the end of the test to make sure.